lib/plugins/package/lib/packageService.js

```diff
@@ -225,10 +225,16 @@ module.exports = {
     params.include.forEach((pattern) => {
       patterns.push(pattern);
     });
+
+    // This strips "./" from the beginning of an includes pattern to avoid
+    // duplication of paths. See https://github.com/serverless/serverless/issues/5748
+    const strippedIncludes = params.include.map((pattern) => {
```

Comment:
I think relying on [`path.normalize`](https://nodejs.org/api/path.html#path_path_normalize_path) would be more reliable (it'll also pick other weird duplication cases):

```javascript
const { normalize } = require('path')

normalize('./bin'); // 'bin'
normalize('bin'); // 'bin'
normalize('./foo/../bin'); // 'bin'
```